shipstation/api.py

```diff
@@ -398,3 +417,37 @@ def post(self, endpoint='', data=None):
         )
         if self.debug:
             pprint.PrettyPrinter(indent=4).pprint(r.json())
+
+    def fetch_orders(self, parameters={}):
+        """
+            Query and fetch existing orders from ShipStation
+
+            Args:
+                parameters (dict): Dict of filters to filter by.
+
+            Returns: JSON string of the returned matching orders.
```

Comment:
```suggestion
```

I don't see a return value here, just a side-effect of calling `self.get`. This should probably be removed.